CONTRIBUTING.md

```diff
@@ -0,0 +1,52 @@
+<!--
+
+    Copyright (c) 2019-present Sonatype, Inc.
+    This program and the accompanying materials are made available under
+    the terms of the Eclipse Public License 2.0 which accompanies this
+    distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+
+-->
+
+# How to be a contributor to this project
+
+## Talk to a human first
+
+### For Sonatypers
+
+Looking to dive in? FANTASTIC! Reach out to one of our experts and we can help get you started in the right direction.
+
+* Hit us up on our Slack channel #integrations.
+* Please fill out an issue (Jira INT project) for your PR so that we have traceability as to what you are trying to fix,
+  versus how you fixed it.
+
+### For Non-Sonatypers
+
+* Come hang out with us at our [gitter channel](https://gitter.im/sonatype/nexus-developers) so we can try to understand the problem you are trying to solve.
+* Sign the [Sonatype CLA](https://sonatypecla.herokuapp.com/sign-cla)
+
+## Submitting a PR
+
+* For Sonatype submitters, add the `@sonatype/integrations` group as a reviewer on the PR
+* Try to fix one thing per pull request! Many people work on this code, so the more focused your changes are, the less
+  of a headache other people will have when they merge their work in.
+* Ensure your Pull Request passes tests either locally or via Jenkins (it will run automatically on your PR)
+* If you're stuck, ask our [gitter channel](https://gitter.im/sonatype/nexus-developers) or #integrations in Slack! There are a number of
+  experienced programmers who are happy to help with learning and troubleshooting.
+
+### PR commenting protocol
+
+Here are the general rules to follow when commenting on PRs for this repo:
+
+* When leaving comments on a PR, leave them all as individual comments as opposed to "Starting a Review".  This is
+  because some people prefer to receive each comment as a separate email.
```

Comment:
Actually I agree with @fernau... I actually really dislike this. :shrug: on this repo as there will be very few contributions I expect.

Reply:
> 
> 
> Actually I agree with @fernau... I actually really dislike this. 🤷 on this repo as there will be very few contributions I expect.

Agreed, removed this.